bites_test.py

```diff
@@ -177,9 +198,9 @@ class InitialiseAnthillTest(unittest.TestCase):
   def test_can_initialise_anthill_as_list_with_len_five_and_every_element_is_None(self):
   # test 27
     ants = ['purple', 'red', 'brown', 'yellow', 'green']
+    bites_game = Bites(ants, TOKENS_FOR_TRAIL)
```

Comment:
See comment above regarding not using the constants. I realise you need the tokens in order to initialise the Bites class, but since you're not testing the trail here, you could use `tokens_for_trail = {}` an empty dictionary as the tokens and this test should still pass I expect.